src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -32,6 +32,7 @@ import styles                             from 'dashboard/Data/Browser/Browser.s
 import subscribeTo                        from 'lib/subscribeTo';
 import * as ColumnPreferences             from 'lib/ColumnPreferences';
 import * as queryString                   from 'query-string';
+import DocumentTitle                      from 'react-document-title';
```

Comment:
I’m always cautious when adding new dependencies.

Did you consider Helmet? It is well maintained and popular.